### DISCUSSION TITLE
New line highlight mode: "dimming"

### DISCUSSION THREAD

**Discussion by taintedcypher** [KevinBatdorf/code-block-pro#292] [Ideas]
I played with the CSS for the line highlighting and wanted to share this, maybe others will find it useful?
This is what it looks like:
![line-highlight](https://github.com/KevinBatdorf/code-block-pro/assets/119351153/77737172-e70b-4445-94b6-4bfa5a9f8a14)

It's basically the "blur" method of highlighting, without the actual blur, and the opacity of the non-highlighted lines bumped up from 0.3 to 0.5. I personally find it slightly more readable then the two default options.

What do you think? Could it be added to the extension?

**Comment by KevinBatdorf** [comment]
Looks nice! I could probably make it more customizable or some options to select from. Is it just opacity on the other lines?

hard to tell whether I think it looks nice though without seeing a full page of text and the code in the context. It's often done this way to draw your eyes to whatever the paragraph above it just described.

**Reply by taintedcypher** [reply]
It's the code for the blur highlight, so there might be something else at play than the opacity! I just overrode the blur and opacity rules :)
Later, I also played with saturation, that make for some cool effects as well! So many possibilities! Thanks for making this plugin, it's good!
